style.css

```diff
@@ -1 +1,345 @@
+  top: -8px;
+  right: 9px;
+  cursor: pointer;
+  opacity: 0;
+  z-index: 2;
+}
+
+.menu-box {
+  padding-right: 15px;
+}
+
+.nav-menu .hamburger {
+  display: block;
+  position: relative;
+  width: 33px;
+  height: 4px;
+  margin-bottom: 5px;
+  background-color: #ddbc8e;
+  border-radius: 3px;
+  z-index: 1;
+  transform-origin: 4px 0px;
+  transition: transform 0.5s cubic-bezier(0.77, 0.2, 0.05, 1),
+    background-color 0.5s cubic-bezier(0.77, 0.2, 0.05, 1), opacity 0.55s ease;
+}
+
+.nav-menu span::before {
+  transform-origin: 0% 0%;
+}
+
+.nav-menu span::after {
+  transform-origin: 0% 100%;
+}
+
+.nav-menu-list {
+  list-style: none;
+  text-align: center;
+  position: absolute;
+  width: 320px;
+  margin: 30px -50px 0px 0px;
+  padding: 5px;
+  padding-top: 50px;
+  background-color: cornsilk;
+  transform-origin: 0% 0%;
+  transform: translate(-100%, 0);
+  transition: transform 300ms cubic-bezier(0.5, 0, 0.5, 1);
+}
+
+.nav-menu-list li {
+  font-size: 25px;
+  padding-top: 40px;
+  padding-right: 0px;
+  padding-bottom: 40px;
+}
+
+.nav-menu input:checked ~ ul {
+  transform: translateX(100%);
+}
+
+.menu-link {
+  text-decoration: none;
+  color: olivedrab;
+}
+
+.menu-link:hover {
+  padding-left: 50px;
+  color: brown;
+}
+
+.text-box {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding-top: 15%;
+}
+
+h2 {
+  padding-top: 10px;
+  margin-bottom: 15px;
+  font-size: 21px;
+  color: #4285f4;
+  text-transform: uppercase;
+}
+
+p {
+  margin-top: 10px;
+  color: rgb(65, 47, 4);
+  font-size: 18px;
+  font-style: oblique;
+  width: 80%;
+}
+
+.text-box p {
+  text-align: center;
+}
+
+.main-image {
+  width: 414px;
+  height: 500px;
+}
+
+.collections {
+  display: flex;
+}
+
+.img-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.collections-img {
+  width: 190px;
+  height: 207px;
+  padding: 4px;
+}
+.credit {
+  font-size: 10px;
+  color: rgb(255, 255, 255);
+}
+
+h3 {
+  text-align: center;
+  color: #4285f4;
+  font-size: 21px;
+}
+
+hr {
+  width: 90%;
+  margin-top: 20px;
+  padding: 0.3px;
+  border: none;
+  background-color: rgb(65, 47, 4);
+  opacity: 0.1;
+}
+
+.sitemap-container ul {
+  padding: 15px 15px 15px 40px;
+  background-color: #f3e0dc;
+}
+
+.sitemap-container li {
+  list-style: none;
+}
+
+.sitemap-container a {
+  text-decoration: none;
+  color: rgb(189, 150, 52);
+}
+
+.sitemap-link:hover {
+  color: rgb(255, 255, 255);
+  cursor: pointer;
+}
+
+@media (min-width: 600px) {
+  header {
+    justify-content: center;
+  }
+
+  h1 {
+    font-size: 30px;
+  }
+
+  div h1 {
+    padding-left: 0px;
+  }
+  .nav-menu {
+    padding-right: 10px;
+  }
+  h2 {
+    padding-top: 0px;
+    margin-bottom: 25px;
+    font-size: 30px;
+  }
+
+  p {
+    font-size: 26px;
+    font-style: normal;
+  }
+
+  h3 {
+    font-size: 28px;
+  }
+
+  .main-image {
+    width: 800px;
+    height: 900px;
```

Comment:
Your image is 800px width, but this is set for screen width 600px and above. Which means when the screen is at 600px (all the way up to 800px) width, the image is wider than the screen. 
You could set the image to `width: 100%` instead so it adapts to the screen width.